### DISCUSSION TITLE
Customize Tabufline's "tab" module

### DISCUSSION THREAD

**Discussion by realhackcraft** [NvChad/NvChad#2936] [Q&A] [answered]
On the screenshot in the main page, the tabs module of tabufline looked like this:
![Screenshot 2024-07-19 at 10 00 22](https://github.com/user-attachments/assets/46be9d5e-139d-48aa-a4e6-ba1ecfee55a2)

How can I customize mine to look the same, with the "TABS" label and a x button?

This is mine:

![Screenshot 2024-07-19 at 10 01 24](https://github.com/user-attachments/assets/fd02f2e0-73ef-47d7-9904-bbba32c84790)

My config:
```lua
  tabufline = {
    enabled = true,
    lazyload = true,
    order = { "treeOffset", "buffers", "tabs", "btns" },
    modules = nil,
  }
```

**Comment by siduck** [comment, marked as answer]
only option is that you need to copy default tabs module, and then rename on it, i removed the tabs text as its long and the cross symbol  as the close icon is enough , over right
